src/IoTHub.Portal.Shared/Models/v1.0/ActionModel.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace IoTHub.Portal.Shared.Models.v10
+{
+    public class ActionModel
+    {
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public string RoleId { get; set; }
+    }
```

Comment:
This class is useless

Reply:
Yes, this class does'nt exist anymore with the new architecture